patcher/main.go

```diff
@@ -26,20 +26,10 @@ var protocol = flag.String("protocol", "udp", fmt.Sprintf("network protocol to u
 
 func main() {
 	flag.Parse()
-
-	logFile, err := os.Create("game.log")
```

Comment:
Moved to `func runClient()` , changed from os.Create to os.Open, using APPEND and CREATE mode as to not overwrite existing log